todo_app/app.py

```diff
@@ -12,3 +14,9 @@
 def index():
     items = get_items()
     return render_template("index.html", items = items)
+
+@app.route('/add', methods=['POST'])
+def add():
+    form_value = request.form["todo"]
+    todo = add_item(title)
```

Comment:
You're _really_ close here, there are two issues:
1) a mismatch of variable names - you read the user's input out into the `form_value` variable but then you try to use `title` to pass to the `add_item` function, so we want to switch this to:
```suggestion
    title = request.form["todo"]
    todo = add_item(title)
```

And then we just need to say what should happen after the user has successfully added an item. We can do that with the `redirect` function that you've imported:
```
    return redirect('/')
```